test/inputs.test.ts

```diff
@@ -0,0 +1,148 @@
+import { checkSpreadsheetStatus, fetchRedirectRows, mergeHeaders } from '../src/inputs';
+import { fetchMock, setupGlobal } from './setupJest';
+
+setupGlobal();
+
+describe('inputs', () => {
+  const expectedLookup =
+    'https://example.org/keymaster/values/Redirects!A:E?key=gozarian&valueRenderOption=UNFORMATTED_VALUE';
```

Comment:
This URI is computed by hand so we don't accidently make our test meet our implementation (instead of the other way around). The replacement parameters that got hard-inlined here match those in the setupJest.ts file.